Resources/Textures/Objects/Decoration/Flora/flora_shadow_trees.rsi/meta.json

```diff
@@ -0,0 +1,29 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/e00cae8d065f9cf520688cc0dd0e15ba5bef12a9 and recolor by TheShuEd",
+    "size": {
```

Comment:
Needs an up to date commit that was used to get.

Reply:
https://github.com/tgstation/tgstation/blob/e00cae8d065f9cf520688cc0dd0e15ba5bef12a9/icons/obj/flora/jungleflora.dmi
this link?

Reply:
yeah that looks good.